build/builder.sh go generate ./... doesn't call glock, errors out

`make protobuf` hits this error:
```
cd ./cockroach/build && npm install --no-progress
npm WARN package.json @ No repository field.
npm ERR! git clone git@github.com:github:sintef-9012/Proto2TypeScript Cloning into bare repository '/root/.npm/_git-remotes/git-github-com-github-sintef-9012-Proto2TypeScript-0d69e939'...
npm ERR! git clone git@github.com:github:sintef-9012/Proto2TypeScript fatal: remote error: 
npm ERR! git clone git@github.com:github:sintef-9012/Proto2TypeScript    is not a valid repository name
npm ERR! git clone git@github.com:github:sintef-9012/Proto2TypeScript   Email support@github.com for help
npm ERR! notarget No compatible version found: proto2ts@'github:sintef-9012/Proto2TypeScript'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["1.0.0","2.0.0"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! System Linux 3.16.0-4-amd64
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "--no-progress"
npm ERR! cwd /go/src/github.com/cockroachdb/cockroach/build
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.21
npm ERR! code ETARGET
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /go/src/github.com/cockroachdb/cockroach/build/npm-debug.log
npm ERR! not ok code 0
cockroach/build/protobuf.mk:91: recipe for target 'cockroach/build/npm.installed' failed
make[1]: *** [cockroach/build/npm.installed] Error 1
make[1]: Leaving directory '/go/src/github.com/cockroachdb'
Makefile:208: recipe for target 'protobuf' failed
make: *** [protobuf] Error 2
```

It works if I make this change to `proto2ts`:
```
-    "proto2ts": "github:sintef-9012/Proto2TypeScript",
+    "proto2ts": "sintef-9012/Proto2TypeScript",
```

This is on a Debian 8.1.

By the way, I had to install `npm` and `nodejs-legacy` (the latter wasn't easy to identify..). These should be added to the prerequisites in CONTRIBUTING.md

@tamird

Not much luck running `build/builder.sh go generate ./...` either:

```
make -C .. -f cockroach/build/protobuf.mk
make[1]: Entering directory '/go/src/github.com/cockroachdb'
make[1]: *** No rule to make target '/go/bin/protoc', needed by 'cockroach/acceptance/cluster/testconfig.pb.go'.  Stop.
make[1]: Leaving directory '/go/src/github.com/cockroachdb'
Makefile:208: recipe for target 'protobuf' failed
make: *** [protobuf] Error 2
main.go:19: running "make": exit status 2
cli/context.go:69: running "stringer": exec: "stringer": executable file not found in $PATH
roachpb/method.go:22: running "stringer": exec: "stringer": executable file not found in $PATH
security/securitytest/securitytest.go:27: running "go-bindata": exec: "go-bindata": executable file not found in $PATH
sql/session.go:99: running "stringer": exec: "stringer": executable file not found in $PATH
Type checking sql.y
Compiling sql.go
awk -f all_keywords.awk < sql.y > keywords.go
gofmt -s -w keywords.go
awk -f reserved_keywords.awk < sql.y > reserved_keywords.go
gofmt -s -w reserved_keywords.go
sql/parser/parse.go:52: running "stringer": exec: "stringer": executable file not found in $PATH
sql/pgwire/types.go:39: running "stringer": exec: "stringer": executable file not found in $PATH
sql/privilege/privilege.go:25: running "stringer": exec: "stringer": executable file not found in $PATH
sql/sqlbase/structured.go:51: running "stringer": exec: "stringer": executable file not found in $PATH
make -C /go/src/github.com/cockroachdb/cockroach/.. -f cockroach/build/protobuf.mk
make[1]: Entering directory '/go/src/github.com/cockroachdb'
make[1]: *** No rule to make target '/go/bin/protoc', needed by 'cockroach/acceptance/cluster/testconfig.pb.go'.  Stop.
make[1]: Leaving directory '/go/src/github.com/cockroachdb'
Makefile:52: recipe for target 'protobuf' failed
make: *** [protobuf] Error 2
ui/ui.go:21: running "make": exit status 2
util/encoding/encoding.go:836: running "stringer": exec: "stringer": executable file not found in $PATH
```
I do have protoc and stringer in my go bin dir:
```
# echo $GOPATH
/go
# ls -la /go/bin/protoc /go/bin/stringer
-rwxr-xr-x 1 root root 56337480 Jun 20 15:00 /go/bin/protoc
-rwxr-xr-x 1 root root  7328152 Jun 14 17:29 /go/bin/stringer
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -26,8 +26,8 @@ regenerate the associated `.pb.{go,cc,h}` or `.js` files using `go generate
 
 We advise to run `go generate` using our embedded Docker setup.
 `build/builder.sh` is a wrapper script designed to make this convenient. You can
-run `build/builder.sh go generate ./...` from the repository root to get the
-intended result.
+run `build/builder.sh env SKIP_BOOTSTRAP=0 go generate ./...` from the repository
+root to get the intended result.
 
 If you want to run it outside of Docker, `go generate` requires a collection of
 Node.js modules which are installed via npm.

